plugin/path_static_account_secrets.go

```diff
@@ -0,0 +1,110 @@
+package gcpsecrets
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/sdk/framework"
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func pathStaticAccountSecretServiceAccountKey(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern:    fmt.Sprintf("%s/%s/key", staticAccountPathPrefix, framework.GenericNameRegex("static_account")),
+		Deprecated: true,
+		Fields: map[string]*framework.FieldSchema{
+			"static_account": {
```

Comment:
I suggest to replace `static_account` here with `name` along with what's in `GenericNameRegex()` above. This is more consistent with what's in `pathStaticAccount`. This also applies to `pathStaticAccountSecretAccessToken`.

Reply:
I have renamed this.